cookbook/security/custom_authentication_provider.rst

```diff
@@ -484,30 +497,48 @@ You are finished! You can now define parts of your app as under WSSE protection.
 
     .. code-block:: yaml
 
+        # app/config/security.yml
         security:
+            # ...
+
             firewalls:
                 wsse_secured:
-                    pattern:   /api/.*
+                    pattern:   ^/api/
                     stateless: true
                     wsse:      true
 
     .. code-block:: xml
 
-        <config>
-            <firewall name="wsse_secured" pattern="/api/.*">
-                <stateless />
-                <wsse />
-            </firewall>
-        </config>
+        <!-- app/config/security.xml -->
+        <?xml version="1.0" encoding="UTF-8"?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+            <config>
+                <!-- ... -->
+
+                <firewall
+                    name="wsse_secured"
+                    pattern="^/api/"
+                    stateless="true"
+                    wsse="true" />
```

Comment:
when putting attributes on new lines, I would love to put `/>` on its own line too (at the same level as the start `<firewall`). It'll bring the eye back to the correct level (and looks similair to multi-line PHP arrays where `),` is put on its own line as well)